karma.conf.js

```diff
@@ -29,11 +59,7 @@ module.exports = function karma(config) {
     reporters: ['dots'],
 
     files: [
-      'test/*.{jsx,js}',
-    ],
-
-    exclude: [
-      'test/_*.{jsx,js}',
+      'test/*-spec.{jsx,js}',
```

Comment:
Just to be clear, you wanted to revert changes to the `files` and `exclude` properties right? I've changed it back to what it was originally.